go get -v github.com/justwatchcom/gopass fails build

### Summary
When updating gopass with go get go build fails.

go version: 1.10.2 
gopass version: v1.7.2

upgrading from:
gopass version  1.6.10 via go 1.9.2

GOOS: Darwin  amd64

Go build portion of get fails with the following error:
`go get -v -u github.com/justwatchcom/gopass
github.com/justwatchcom/gopass (download)
github.com/gopasspw/gopass (download)
github.com/gopasspw/gopass/vendor/github.com/hashicorp/golang-lru/simplelru
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/cast5
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/poly1305
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/curve25519
github.com/gopasspw/gopass/pkg/tree
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/salsa20/salsa
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/openpgp/errors
github.com/gopasspw/gopass/vendor/golang.org/x/text/encoding/internal/identifier
github.com/gopasspw/gopass/vendor/rsc.io/qr/gf256
github.com/gopasspw/gopass/vendor/github.com/hashicorp/hcl/hcl/strconv
github.com/gopasspw/gopass/pkg/protect
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/nacl/secretbox
github.com/justwatchcom/gopass/vendor/github.com/mattn/go-isatty
github.com/gopasspw/gopass/vendor/github.com/mattn/go-isatty
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/openpgp/armor
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/openpgp/s2k
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/blake2b
github.com/gopasspw/gopass/vendor/golang.org/x/net/context
github.com/gopasspw/gopass/vendor/github.com/mattn/go-colorable
github.com/justwatchcom/gopass/vendor/github.com/mattn/go-colorable
github.com/justwatchcom/gopass/vendor/github.com/blang/semver
github.com/gopasspw/gopass/vendor/github.com/blang/semver
github.com/gopasspw/gopass/pkg/ctxutil
github.com/gopasspw/gopass/vendor/github.com/pkg/errors
github.com/gopasspw/gopass/vendor/github.com/cenkalti/backoff
github.com/gopasspw/gopass/vendor/github.com/fatih/color
github.com/justwatchcom/gopass/vendor/github.com/fatih/color
github.com/gopasspw/gopass/pkg/fsutil
github.com/gopasspw/gopass/vendor/github.com/mitchellh/go-homedir
github.com/gopasspw/gopass/pkg/backend/crypto/gpg
github.com/gopasspw/gopass/vendor/github.com/hashicorp/golang-lru
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/openpgp/elgamal
github.com/gopasspw/gopass/pkg/out
github.com/gopasspw/gopass/vendor/github.com/alecthomas/binary
github.com/gopasspw/gopass/pkg/backend
github.com/gopasspw/gopass/vendor/github.com/golang/protobuf/proto
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/openpgp/packet
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/argon2
github.com/gopasspw/gopass/pkg/backend/crypto/plain
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/nacl/box
github.com/gopasspw/gopass/pkg/agent/client
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/sha3
github.com/gopasspw/gopass/pkg/store
github.com/gopasspw/gopass/vendor/github.com/sergi/go-diff/diffmatchpatch
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-billy.v4
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing
github.com/gopasspw/gopass/vendor/github.com/src-d/gcfg/token
github.com/gopasspw/gopass/vendor/github.com/src-d/gcfg/types
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-billy.v4/helper/polyfill
github.com/gopasspw/gopass/pkg/backend/rcs/git/cli
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-billy.v4/util
github.com/gopasspw/gopass/vendor/github.com/src-d/gcfg/scanner
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-billy.v4/helper/chroot
github.com/gopasspw/gopass/vendor/gopkg.in/warnings.v0
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/internal/revision
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/filemode
github.com/gopasspw/gopass/vendor/github.com/src-d/gcfg
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-billy.v4/osfs
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/format/gitignore
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/utils/binary
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/cache
github.com/gopasspw/gopass/vendor/github.com/jbenet/go-context/io
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/format/index
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/format/idxfile
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/utils/ioutil
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/format/config
github.com/gopasspw/gopass/vendor/github.com/emirpasic/gods/utils
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/format/diff
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/utils/diff
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/config
github.com/gopasspw/gopass/vendor/github.com/emirpasic/gods/containers
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/storer
github.com/gopasspw/gopass/vendor/golang.org/x/text/transform
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/format/pktline
github.com/gopasspw/gopass/vendor/github.com/emirpasic/gods/lists
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/protocol/packp/capability
github.com/gopasspw/gopass/vendor/github.com/emirpasic/gods/trees
github.com/gopasspw/gopass/vendor/github.com/emirpasic/gods/lists/arraylist
github.com/gopasspw/gopass/vendor/golang.org/x/text/unicode/norm
github.com/gopasspw/gopass/vendor/github.com/pelletier/go-buffruneio
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/protocol/packp/sideband
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/format/packfile
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/storage
github.com/gopasspw/gopass/vendor/github.com/emirpasic/gods/trees/binaryheap
github.com/gopasspw/gopass/vendor/github.com/kevinburke/ssh_config
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/ed25519/internal/edwards25519
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/internal/chacha20
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/storage/memory
github.com/gopasspw/gopass/pkg/backend/rcs/noop
github.com/gopasspw/gopass/pkg/backend/storage/fs
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/protocol/packp
github.com/gopasspw/gopass/vendor/github.com/hashicorp/go-cleanhttp
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/ed25519
github.com/gopasspw/gopass/vendor/github.com/hashicorp/go-rootcerts
github.com/gopasspw/gopass/vendor/github.com/hashicorp/serf/coordinate
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/ssh
github.com/gopasspw/gopass/pkg/backend/storage/kv/inmem
github.com/gopasspw/gopass/vendor/github.com/hashicorp/consul/api
github.com/gopasspw/gopass/pkg/config/secrets
github.com/gopasspw/gopass/vendor/gopkg.in/yaml.v2
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/openpgp/clearsign
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/openpgp
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/transport
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/format/objfile
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/transport/internal/common
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/storage/filesystem/internal/dotgit
github.com/gopasspw/gopass/pkg/backend/crypto/gpg/cli
github.com/gopasspw/gopass/pkg/backend/crypto/gpg/openpgp
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/transport/git
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/utils/merkletrie/noder
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/storage/filesystem
github.com/gopasspw/gopass/pkg/backend/crypto/xc/xcpb
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/utils/merkletrie/internal/frame
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/transport/http
github.com/gopasspw/gopass/pkg/backend/crypto/xc/keyring
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/utils/merkletrie
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/object
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/utils/merkletrie/filesystem
github.com/gopasspw/gopass/pkg/backend/crypto/xc
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/utils/merkletrie/index
github.com/gopasspw/gopass/pkg/tree/simple
github.com/gopasspw/gopass/vendor/golang.org/x/sys/unix
github.com/gopasspw/gopass/vendor/github.com/xrash/smetrics
github.com/gopasspw/gopass/vendor/github.com/atotto/clipboard
github.com/gopasspw/gopass/vendor/github.com/urfave/cli
github.com/gopasspw/gopass/vendor/github.com/muesli/crunchy
github.com/gopasspw/gopass/pkg/tempfile
github.com/gopasspw/gopass/vendor/github.com/kballard/go-shellquote
github.com/gopasspw/gopass/vendor/golang.org/x/text/encoding
github.com/gopasspw/gopass/vendor/github.com/gdamore/tcell/terminfo
github.com/gopasspw/gopass/vendor/github.com/gdamore/encoding
github.com/gopasspw/gopass/vendor/github.com/lucasb-eyer/go-colorful
github.com/gopasspw/gopass/pkg/backend/storage/kv/consul
github.com/gopasspw/gopass/vendor/github.com/mattn/go-runewidth
github.com/gopasspw/gopass/pkg/hashsum
github.com/gopasspw/gopass/pkg/hibp/api
github.com/gopasspw/gopass/vendor/github.com/nsf/termbox-go
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/revlist
github.com/gopasspw/gopass/pkg/hibp/dump
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/ssh/terminal
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/transport/server
github.com/gopasspw/gopass/vendor/github.com/jroimartin/gocui
github.com/gopasspw/gopass/pkg/pwgen
github.com/gopasspw/gopass/vendor/github.com/fatih/structs
github.com/gopasspw/gopass/pkg/store/secret
github.com/gopasspw/gopass/vendor/github.com/muesli/goprogressbar
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/transport/file
github.com/gopasspw/gopass/pkg/termio
github.com/gopasspw/gopass/vendor/github.com/hashicorp/errwrap
github.com/gopasspw/gopass/vendor/github.com/gdamore/tcell
github.com/gopasspw/gopass/vendor/github.com/hashicorp/hcl/hcl/token
github.com/gopasspw/gopass/vendor/github.com/ryanuber/go-glob
github.com/gopasspw/gopass/vendor/github.com/golang/snappy
github.com/gopasspw/gopass/vendor/github.com/mitchellh/mapstructure
github.com/gopasspw/gopass/vendor/github.com/hashicorp/vault/helper/strutil
github.com/gopasspw/gopass/vendor/github.com/hashicorp/go-multierror
github.com/gopasspw/gopass/pkg/completion/fish
github.com/gopasspw/gopass/pkg/completion/zsh
github.com/gopasspw/gopass/vendor/github.com/hashicorp/hcl/hcl/ast
github.com/gopasspw/gopass/pkg/editor
github.com/gopasspw/gopass/vendor/github.com/hashicorp/hcl/hcl/scanner
github.com/gopasspw/gopass/vendor/github.com/hashicorp/hcl/json/token
github.com/gopasspw/gopass/vendor/github.com/hashicorp/vault/helper/compressutil
github.com/gopasspw/gopass/vendor/github.com/hashicorp/vault/helper/parseutil
github.com/gopasspw/gopass/vendor/github.com/sethgrid/pester
github.com/gopasspw/gopass/vendor/github.com/hashicorp/hcl/json/scanner
github.com/gopasspw/gopass/vendor/github.com/hashicorp/vault/helper/jsonutil
github.com/gopasspw/gopass/vendor/github.com/hashicorp/hcl/hcl/parser
github.com/gopasspw/gopass/vendor/golang.org/x/net/http2/hpack
github.com/gopasspw/gopass/vendor/github.com/hashicorp/hcl/json/parser
github.com/gopasspw/gopass/vendor/golang.org/x/net/publicsuffix
github.com/gopasspw/gopass/vendor/golang.org/x/text/unicode/bidi
github.com/gopasspw/gopass/pkg/jsonapi/manifest
github.com/gopasspw/gopass/vendor/rsc.io/qr/coding
github.com/gopasspw/gopass/vendor/github.com/hashicorp/hcl
github.com/gopasspw/gopass/vendor/github.com/martinhoefling/goxkcdpwgen/xkcdpwgen
github.com/gopasspw/gopass/vendor/github.com/skip2/go-qrcode/bitset
github.com/gopasspw/gopass/pkg/termutil
github.com/gopasspw/gopass/pkg/tpl
github.com/gopasspw/gopass/vendor/github.com/skip2/go-qrcode/reedsolomon
github.com/gopasspw/gopass/vendor/github.com/dominikschulz/github-releases/ghrel
github.com/gopasspw/gopass/vendor/github.com/schollz/closestmatch
github.com/gopasspw/gopass/vendor/github.com/skip2/go-qrcode
github.com/gopasspw/gopass/vendor/rsc.io/qr
github.com/gopasspw/gopass/pkg/updater
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/ssh/agent
github.com/gopasspw/gopass/vendor/golang.org/x/crypto/ssh/knownhosts
github.com/gopasspw/gopass/vendor/github.com/gokyle/twofactor
github.com/gopasspw/gopass/vendor/golang.org/x/text/secure/bidirule
github.com/gopasspw/gopass/pkg/qrcon
github.com/gopasspw/gopass/vendor/golang.org/x/net/idna
github.com/gopasspw/gopass/pkg/otp
github.com/gopasspw/gopass/pkg/pinentry
github.com/justwatchcom/gopass/vendor/github.com/urfave/cli
github.com/gopasspw/gopass/pkg/agent
github.com/justwatchcom/gopass/vendor/golang.org/x/sys/unix
github.com/gopasspw/gopass/vendor/github.com/xanzy/ssh-agent
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/transport/ssh
github.com/gopasspw/gopass/vendor/golang.org/x/net/lex/httplex
github.com/gopasspw/gopass/vendor/github.com/gdamore/tcell/termbox
github.com/gopasspw/gopass/vendor/golang.org/x/net/http2
github.com/gopasspw/gopass/pkg/termwiz
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4/plumbing/transport/client
github.com/gopasspw/gopass/pkg/cui
github.com/gopasspw/gopass/vendor/gopkg.in/src-d/go-git.v4
github.com/justwatchcom/gopass/vendor/golang.org/x/crypto/ssh/terminal
github.com/gopasspw/gopass/pkg/pwgen/xkcdgen
github.com/gopasspw/gopass/pkg/backend/rcs/git/gogit
github.com/gopasspw/gopass/pkg/store/sub
github.com/gopasspw/gopass/vendor/github.com/hashicorp/vault/api
github.com/gopasspw/gopass/pkg/config
github.com/gopasspw/gopass/pkg/notify
github.com/gopasspw/gopass/pkg/store/vault
github.com/gopasspw/gopass/pkg/audit
github.com/gopasspw/gopass/pkg/clipboard
github.com/gopasspw/gopass/pkg/store/root
github.com/gopasspw/gopass/pkg/jsonapi
github.com/gopasspw/gopass/pkg/action
github.com/gopasspw/gopass/pkg/action/xc
github.com/gopasspw/gopass/pkg/action/create
github.com/gopasspw/gopass/pkg/action/binary
github.com/justwatchcom/gopass
# github.com/justwatchcom/gopass
go/src/github.com/justwatchcom/gopass/app.go:35:23: cannot use sv (type "github.com/justwatchcom/gopass/vendor/github.com/blang/semver".Version) as type "github.com/gopasspw/gopass/vendor/github.com/blang/semver".Version in argument to action.New
go/src/github.com/justwatchcom/gopass/app.go:58:18: cannot use c (type *"github.com/justwatchcom/gopass/vendor/github.com/urfave/cli".Context) as type *"github.com/gopasspw/gopass/vendor/github.com/urfave/cli".Context in argument to action.Complete
go/src/github.com/justwatchcom/gopass/app.go:63:41: cannot use c (type *"github.com/justwatchcom/gopass/vendor/github.com/urfave/cli".Context) as type *"github.com/gopasspw/gopass/vendor/github.com/urfave/cli".Context in argument to action.JSONAPI
go/src/github.com/justwatchcom/gopass/app.go:66:47: cannot use c (type *"github.com/justwatchcom/gopass/vendor/github.com/urfave/cli".Context) as type *"github.com/gopasspw/gopass/vendor/github.com/urfave/cli".Context in argument to action.Initialized
go/src/github.com/justwatchcom/gopass/app.go:71:38: cannot use c (type *"github.com/justwatchcom/gopass/vendor/github.com/urfave/cli".Context) as type *"github.com/gopasspw/gopass/vendor/github.com/urfave/cli".Context in argument to action.Show
go/src/github.com/justwatchcom/gopass/app.go:73:37: cannot use c (type *"github.com/justwatchcom/gopass/vendor/github.com/urfave/cli".Context) as type *"github.com/gopasspw/gopass/vendor/github.com/urfave/cli".Context in argument to action.List
go/src/github.com/justwatchcom/gopass/commands.go:61:82: cannot use c (type *"github.com/justwatchcom/gopass/vendor/github.com/urfave/cli".Context) as type *"github.com/gopasspw/gopass/vendor/github.com/urfave/cli".Context in argument to action.Initialized
go/src/github.com/justwatchcom/gopass/commands.go:63:40: cannot use c (type *"github.com/justwatchcom/gopass/vendor/github.com/urfave/cli".Context) as type *"github.com/gopasspw/gopass/vendor/github.com/urfave/cli".Context in argument to action.Audit
go/src/github.com/justwatchcom/gopass/commands.go:83:84: cannot use c (type *"github.com/justwatchcom/gopass/vendor/github.com/urfave/cli".Context) as type *"github.com/gopasspw/gopass/vendor/github.com/urfave/cli".Context in argument to action.Initialized
go/src/github.com/justwatchcom/gopass/commands.go:85:41: cannot use c (type *"github.com/justwatchcom/gopass/vendor/github.com/urfave/cli".Context) as type *"github.com/gopasspw/gopass/vendor/github.com/urfave/cli".Context in argument to action.HIBP
go/src/github.com/justwatchcom/gopass/commands.go:85:41: too many errors`


### Steps To Reproduce
go get -v -u github.com/justwatchcom/gopass

### Expected behavior
Expected gopass to build and be installed to go/bin

### Environment
- OS: Mac OS X High Sierra AMD 64
- OS version: Darwin <redacted> 16.7.0 Darwin Kernel Version 16.7.0: Tue Jan 30 11:27:06 PST 2018; root:xnu-3789.73.11~1/RELEASE_X86_64 x86_64
- gopass Version: 1.7.2 built with go 1.10.2 (also tried with 1.9.2)
- Installation method: from source

--- a/pkg/updater/update.go
+++ b/pkg/updater/update.go
@@ -33,7 +33,7 @@ var (
 )
 
 const (
-	gitHubOrg  = "justwatchcom"
+	gitHubOrg  = "gopasspw"
 	gitHubRepo = "gopass"
 )
 

